src/test/unit/rx_ranges_unittest.cc

```diff
@@ -37,6 +37,18 @@ extern "C" {
 #include "gtest/gtest.h"
 
 extern "C" {
+
+PG_REGISTER_WITH_RESET_TEMPLATE(flight3DConfig_t, flight3DConfig, PG_MOTOR_3D_CONFIG, 0);
+PG_RESET_TEMPLATE(flight3DConfig_t, flight3DConfig,
```

Comment:
Are the tests actually relying on this data? Otherwise, just use `PG_REGISTER()` and don't bother with setting the defaults.

Reply:
No, shouldn't need the data. I originally did test without the RESET and got compilation errors but I forgot to change `PG_REGISTER_WITH_RESET_TEMPLATE` to `PG_REGISTER`. Revised and pushed updates.